components/shared/Table.tsx

```diff
@@ -0,0 +1,75 @@
+import clsx from "clsx";
```

Comment:
I generated this from [shadcn/ui's Table component](https://ui.shadcn.com/docs/components/table). This can be used as the foundation for our other tables going forward as we look to make them more responsive.